src/zcl/definition/cluster.ts

```diff
@@ -4244,6 +4244,19 @@ const Cluster: {
                     {name: 'dpValues', type: BuffaloZclDataType.LIST_TUYA_DATAPOINT_VALUES},
                 ],
             },
+            /**
```

Comment:
Those are sorted by ID, so move it above ID: 6

Reply:
Changed order as requested.